edxadapt/edxadapt.py

```diff
@@ -55,12 +56,24 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
         help=_('Edx Adapt API base URL, e.g. https://edx-adapt.example.com:443/api/v1')
     )
 
+    edx_adapt_course_id = String(
+        default='',
+        display_name=_('Edx Adapt CourseID'),
```

Comment:
CourseID -> Course ID, because it's a regular sentence not a variable name.
And on line 64 too.